src/Kodify/BlogBundle/Resources/views/Post/List/index.html.twig

```diff
@@ -3,7 +3,7 @@
 {% block content %}
 
     {% for post in posts %}
-        <div class="panel panel-default">
+        <div class="panel panel-default panel-home">
```

Comment:
I saw the other solution (https://github.com/natxo-kodify/blog/pull/1#discussion-diff-22247494), but I didn't have time to see the correct use of this: http://terapiaposturalint.wix.com/ricardochacon#!acerca-de-mi/cjg9